client/components/date-picker/content.js

```diff
@@ -16,6 +17,8 @@ import PresetPeriods from './preset-periods';
 import Link from 'components/link';
 import { DateRange } from 'components/calendar';
 
+const isMobileViewport = () => window.innerWidth < 782;
```

Comment:
Is there already an event somewhere that will trigger a re-render on change of window size - or do we need to add logic here to handle that

Reply:
There isn't a listener to trigger re-render on resize, but this calculation is done on every opening of the custom tab as you noted. My opinion is the use of another event listener doesn't justify the benefit of being able to render two calendars when the user is resizing with the custom date tab open. How often will this happen? What do you think @timmyc ?

Reply:
I think we are fine with it as-is. Very much an edge case, and it works fine just the way it is when  you resize the browser.